utils/src/test/scala/za/co/absa/enceladus/utils/types/TypedStructFieldSuite.scala

```diff
@@ -143,17 +146,17 @@ class TypedStructFieldSuite extends FunSuite {
   test("Long type not nullable, with default defined as binary integer") {
     val fieldType = LongType
     val nullable = false
-    val field = createField(fieldType, nullable, Some(-1L)) // scalastyle:ignore null
+    val field = createField(fieldType, nullable, Some(-1L))
     val typed = TypedStructField(field)
     val errMsg = "java.lang.Long cannot be cast to java.lang.String"
     val fail = Failure(new ClassCastException(errMsg))
     checkField(typed, fieldType, fail, fail, nullable, Seq(ValidationError(errMsg)))
   }
 
-  test("Float type nullable, with default defined in exponential notation") {
+  test("Float type nullable, with default defined in exponential notation, allowInfinity is set to true") {
     val fieldType = FloatType
     val nullable = true
-    val field = createField(fieldType, nullable, Some("314e-2")) // scalastyle:ignore null
+    val field = createField(fieldType, nullable, Some("314e-2"), Map( MetadataKeys.allowInfinity->"true" ))
```

Comment:
```suggestion
    val field = createField(fieldType, nullable, Some("314e-2"), Map(MetadataKeys.allowInfinity -> "true"))
```
This looks like some sort of reverse spacing, lol

Reply:
Will fix 😉